checks/check_extra75

```diff
@@ -37,6 +37,10 @@ extra75(){
     if [[ $SECURITYGROUPS == "null" ]]; then
       continue
     fi
```

Comment:
I think this comparison could be removed because the next is the replacement for this, am I right?

We no longer need to compare if `SECURITYGROUPS` object is a null object.

Reply:
I would remove it too.

Reply:
@jfaroagas I went back and forth on removing it and decided to let it be reviewed.

Reply:
Thank you!!